deploy/env/dev.yaml

```diff
@@ -11,4 +11,13 @@ registration-service:
                   "resource": "crt",
                   "clientId": "crt",
                   "public-client": true
-                }'
\ No newline at end of file
+                }'
+  verification.enabled: 'true'
+  verification.daily_limit: '3'
+  verification.attempts_allowed: '5'
+host-operator:
+  registration-service-url: 'https://sso.prod-preview.openshift.io/auth/realms/toolchain-public/protocol/openid-connect/certs'
```

Comment:
This is not the registration-service-url. But since we can't actually provide the reg-service URL for the dev env I would stick with the default value we use in configuration.go and just skip it here.

Reply:
done as part of this: https://github.com/codeready-toolchain/host-operator/pull/263#discussion_r473209798